config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root 'home#index'
+  resources :users
```

Comment:
How do you feel about only declaring the routes you need? This declaration declares all 7, but you're only using `new` and `create`. 

[From the Rails Guide](If your application has many RESTful routes, using :only and :except to generate only the routes that you actually need can cut down on memory use and speed up the routing process.)